.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+##### SUMMARY
+<!--- Describe the change, including rationale and design decisions -->
+
+<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
+
+##### ISSUE TYPE
+
+<!--- Pick one below and delete the other: -->
+- Bug, Docs Fix or other nominal change
+- New or Enhanced Feature
+
+##### Tests
+
+<!--- Document the tests for this change, if any -->
+<!--- See: https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/CONTRIBUTING.md#ci-pipelines -->
+
+<!--- Examples:
+- [] TestDallas - <JobID>
```

Comment:
Question: wouldn't these lines be interpreted by our automation? Because, as far as I know, dci-pipeline reads the full commit message and looks for the keywords defined in auto.conf, so maybe this would cause that dci-pipeline attempts to launch an undersired job
so, maybe, it's better to avoid using TestXXX when reporting the test results in the checklist

Reply:
> wouldn't these lines be interpreted by our automation?

Not at all, because we do regex to match the beginning of the line.

I've done it multiple times before, e.g. #324, #293, #284